### DISCUSSION TITLE
development environment not working as per contribution guidelines

### DISCUSSION THREAD

**Discussion by 609harsh** [opensource9ja/danfojs#659] [General]
I am trying to setup the development environment as per steps given here https://danfo.jsdata.org/contributing-guide#creating-a-development-environment. Getting following error as per image attached.

OS->Ubuntu

![image](https://github.com/javascriptdata/danfojs/assets/97297407/3a4693b8-e50d-474a-b521-fba667ee083c)
![image](https://github.com/javascriptdata/danfojs/assets/97297407/e4e5bbf7-2fe2-45b9-a77f-f3ee94dcb3eb)

**Comment by risenW** [comment]
@609harsh what version of node are you using?
